packages/core/src/components/BaseInput/BaseInput.module.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../styles/Typography";
+
+.wrapper {
+  width: 100%;
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-small);
+  padding-block: var(--spacing-xs);
+  padding-inline: var(--spacing-medium) var(--spacing-xs);
```

Comment:
Let's make it a standard?

Reply:
wdym?

Reply:
Logical properties

Reply:
hmm I agree
Chrome 87+ support
we don't have users with less than 90, but anyway when we upgrade browsers it should be supported